recipes/maxima/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "5.39.0" %}
+{% set name = "maxima" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tgz
+  url: https://sourceforge.net/projects/maxima/files/Maxima-source/{{ version }}-source/{{ name }}-{{ version }}.tar.gz
+  sha256: f58ff98013fcaa37d8661b941672c38db58f982f082d6cfcca1f1b5fac7d71b3
+  patches:
+    # Patches from https://git.sagemath.org/sage.git/tree/build/pkgs/maxima/patches
+    - 0001-taylor2-Avoid-blowing-the-stack-when-diff-expand-isn.patch
+    - infodir.patch
+    - maxima.system.patch
+    - matrixexp.patch
+    - undoing_true_false_printing_patch.patch
```

Comment:
Can you add some comments as to why the patches are needed? We can do that in the feedstock after merging as well.